sdks/java/io/rabbitmq/src/main/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqIO.java

```diff
@@ -82,28 +97,31 @@
  *
  * <p>As for the {@link Read}, the {@link Write} is configured with a RabbitMQ URI.
  *
- * <p>For instance, you can write to an exchange (providing the exchange type):
+ * <p>Examples
  *
  * <pre>{@code
+ * // Publishing to a named, non-durable exchange, declared by Beam:
  * pipeline
  *   .apply(...) // provide PCollection<RabbitMqMessage>
  *   .apply(RabbitMqIO.write().withUri("amqp://user:password@localhost:5672").withExchange("EXCHANGE", "fanout"));
- * }</pre>
  *
- * <p>For instance, you can write to a queue:
+ * // Publishing to an existing exchange
+ * pipeline
+ *   .apply(...) // provide PCollection<RabbitMqMessage>
+ *   .apply(RabbitMqIO.write().withUri("amqp://user:password@localhost:5672").withExchange("EXCHANGE"));
  *
- * <pre>{@code
+ * // Publishing to a named queue in the default exchange:
  * pipeline
  *   .apply(...) // provide PCollection<RabbitMqMessage>
  *   .apply(RabbitMqIO.write().withUri("amqp://user:password@localhost:5672").withQueue("QUEUE"));
- *
  * }</pre>
  */
 @Experimental(Experimental.Kind.SOURCE_SINK)
 public class RabbitMqIO {
   public static Read read() {
     return new AutoValue_RabbitMqIO_Read.Builder()
         .setQueueDeclare(false)
+        .setExchangeDeclare(false)
```

Comment:
I remember I discussed about that when I created the IO. I don't remember why I didn't do that finally. It makes sense anyway, thanks !